hatchet_sdk/hatchet.py

```diff
@@ -99,7 +102,6 @@ def inner(func: Callable[[Context], Any]) -> Callable[[Context], Any]:
         setattr(func, "_step_parents", parents)
         setattr(func, "_step_timeout", timeout)
         setattr(func, "_step_retries", retries)
-        setattr(func, "_step_rate_limits", retries)
```

Comment:
drive by copy-paste issue - didn't cause any issues or anything since it's overridden on the next line